src/OrchardCore.Modules/OrchardCore.Workflows/Trimming/ViewModels/WorkflowTrimmingViewModel.cs

```diff
@@ -0,0 +1,35 @@
+using System;
+using System.Linq;
+using OrchardCore.Workflows.Models;
+
+namespace OrchardCore.Workflows.Trimming.ViewModels;
+
+public class WorkflowTrimmingViewModel
+{
+    public DateTime? LastRunUtc { get; set; }
+
+    public bool Disabled { get; set; }
+
+    public int RetentionDays { get; set; }
+
+    public WorkflowStatusItem[] Statuses { get; set; }
+
+    public WorkflowTrimmingViewModel()
+    {
+        Statuses = Enum.GetValues(typeof(WorkflowStatus))
```

Comment:
This is done in the driver, remove from constructor

Reply:
It's not. This is about filling `Statuses` with all possible `WorkflowStatusItem`s. The driver only marks the selected ones.